backend/src/model/dto/actions.rs

```diff
@@ -35,10 +35,10 @@ pub enum Action {
 #[serde(rename_all = "camelCase")]
 pub struct CreatePlantActionPayload {
     user_id: Uuid,
-    id: Uuid,
+    id: i32,
```

Comment:
Why did you revert all the planting identifiers to `i32`?
The frontend needed those to be a `Uuid`.

For explanation, the interaction is like this:
Create planting with random uuid & add to the store to be immediately visible.
Broadcast this creation to all other clients.
The other clients create the same planting in their respective stores.

If we have integers the optimistic UI is difficult to implement, because another user might create the planting with the same id.

Reply:
Then I'll change it back. Just thought it was weird that all id's are integers but that one is a uuid.